examples/vue-example-app/src/sdk/index.ts

```diff
@@ -1,26 +1,24 @@
-import fetch from 'node-fetch'
-import { ClientBuilder } from '@commercetools/sdk-client-v2'
+import { ClientBuilder } from '@commercetools/ts-client'
 import { createApiBuilderFromCtpClient } from '@commercetools/platform-sdk'
 
-export const projectKey = process.env.VUE_APP_CTP_PROJECT_KEY
+export const projectKey = import.meta.env.VITE_CTP_PROJECT_KEY
 const authMiddlewareOptions = {
   host: 'https://auth.europe-west1.gcp.commercetools.com',
   projectKey,
   credentials: {
-    clientId: process.env.VUE_APP_CTP_CLIENT_ID,
-    clientSecret: process.env.VUE_APP_CTP_CLIENT_SECRET,
+    clientId: import.meta.env.VITE_CTP_CLIENT_ID,
+    clientSecret: import.meta.env.VITE_CTP_CLIENT_SECRET,
   },
   scopes: [`manage_project:${projectKey}`],
-  fetch,
+  httpClient: fetch,
 }
 
 const httpMiddlewareOptions = {
   host: 'https://api.europe-west1.gcp.commercetools.com',
-  fetch,
+  httpClient: fetch,
 }
 
 const client = new ClientBuilder()
-  .withProjectKey(projectKey)
   .withClientCredentialsFlow(authMiddlewareOptions)
   .withHttpMiddleware(httpMiddlewareOptions)
   // .withLoggerMiddleware()
```

Comment:
plese remove the commented lines since it's a pubblic repo or add a note if you think it's good to have